src/transformers/testing_utils.py

```diff
@@ -2431,6 +2432,10 @@ def _device_agnostic_dispatch(device: str, dispatch_table: Dict[str, Callable],
     BACKEND_MANUAL_SEED = {"cuda": torch.cuda.manual_seed, "cpu": torch.manual_seed, "default": torch.manual_seed}
     BACKEND_EMPTY_CACHE = {"cuda": torch.cuda.empty_cache, "cpu": None, "default": None}
     BACKEND_DEVICE_COUNT = {"cuda": torch.cuda.device_count, "cpu": lambda: 0, "default": lambda: 1}
+    if is_torch_xpu_available():
+        BACKEND_MANUAL_SEED["xpu"] = torch.xpu.manual_seed
```

Comment:
this should be removed, more info see https://huggingface.co/docs/transformers/v4.41.3/en/testing#testing-with-a-specific-pytorch-backend-or-device

Reply:
@faaany : thank you for the link. I overlooked it. I removed these lines and used spec.py on my side.